util/pace/util/checkpointer/validation.py

```diff
@@ -118,8 +118,6 @@ def __call__(self, savepoint_name: str, **kwargs: ArrayLike) -> None:
 
             expected = ds[varname][n_calls, self._rank].values
             output = _clip_pace_array_to_target(cast_to_ndarray(array), expected.shape)
-
-            # cannot use relative threshold when comparing to zero value
```

Comment:
Why remove this comment? It explains why we're only testing certain points.

Reply:
This was unintentional, reverting now.